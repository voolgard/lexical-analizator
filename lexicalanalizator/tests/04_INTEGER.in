555 666
111