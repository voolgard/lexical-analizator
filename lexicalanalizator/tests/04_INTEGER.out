1	1	INTEGER	555	555
1	5	INTEGER	666	666
2	1	INTEGER	111	111